model_compression_toolkit/pytorch/graph_substitutions/substitutions/shift_negative_activation.py

```diff
@@ -0,0 +1,145 @@
+                                                           {},
+                                                           input_shape,
+                                                           input_shape,
+                                                           weights={},
+                                                           quantization_attr={},
+                                                           functional_op=operator.add,
+                                                           op_call_kwargs={},
+                                                           op_call_args=[float(add_value)],
+                                                           layer_class=operator.add)
+    return add_node
+
+
+def create_pad_node(next_node_name: str,
+                    prev_node_name: str,
+                    value_to_pad: float,
+                    input_shape: tuple,
+                    pad_top: int,
+                    pad_btm: int,
+                    pad_left: int,
+                    pad_right: int) -> BaseNode:
+    """
+    Create a pad node with a constant value to pad its input tensor.
+
+    Args:
+        next_node_name: Name of the node next to the pad node.
+        prev_node_name: Name of the node previous to the pad node.
+        value_to_pad: Constant to use for padding the input of the node.
+        input_shape: Shape of input tensor.
+        pad_top: Number of elements to pad above the tensor.
+        pad_btm: Number of elements to pad below the tensor.
+        pad_left: Number of elements to pad left to the tensor.
+        pad_right: Number of elements to pad right to the tensor.
+
+    Returns:
+        A pad node which pads its input with a constant value.
+    """
+
+    pad_node_name = next_node_name + '_pre_pad'
+
+    op_call_kwargs = {PAD: [pad_left, pad_right, pad_top, pad_btm],
+               VALUE: float(value_to_pad)}
+
+    padded_shape = input_shape[0]
+    padded_shape[1] += pad_top + pad_btm
+    padded_shape[2] += pad_left + pad_right
+    pad_node = common.graph.functional_node.FunctionalNode(pad_node_name,
+                                                           {},
+                                                           input_shape,
+                                                           tuple(padded_shape),
+                                                           weights={},
+                                                           quantization_attr={},
+                                                           functional_op=torch.nn.functional.pad,
+                                                           op_call_kwargs=op_call_kwargs,
+                                                           op_call_args=[],
+                                                           layer_class=torch.nn.functional.pad)
+    return pad_node
+
+
+def compute_op2d_padding():
+    """
+    Dummy function, needed for API.
+    Returns:
+        None
+    """
+    return None
```

Comment:
Add the following function here (similar to the comments in Keras):
def pytorch_apply_shift_negative_correction( graph,qc):
      return apply_shift_negative_correction(graph, ...)